vehicles/hovercraft.lua

```diff
@@ -0,0 +1,20 @@
+local definition = ...
+local types = {
+    "blue", "green", "red", "yellow"
+}
+
+for _, color in pairs(types) do
+    if minetest.settings:get_bool("vehicle_mash.enable_hover_" .. color) == false then return end
```

Comment:
Shouldn't this be `continue` - or in Lua fashion - an `if` around the entire code block below?